app/services/import/persist-licence.service.js

```diff
@@ -97,10 +98,40 @@ async function _persistLicenceVersionPurpose (trx, updatedAt, licenceVersionPurp
 
 async function _persistLicenceVersionPurposes (trx, updatedAt, licenceVersionPurposes, licenceVersionId) {
   for (const licenceVersionPurpose of licenceVersionPurposes) {
-    await _persistLicenceVersionPurpose(trx, updatedAt, licenceVersionPurpose, licenceVersionId)
+    const licenceVersionPurposeConditions = licenceVersionPurpose.licenceVersionPurposeConditions
+
+    const { id } = await _persistLicenceVersionPurpose(
+      trx, updatedAt, licenceVersionPurpose, licenceVersionId)
+
+    await _persistLicenceVersionPurposesConditions(trx, updatedAt, licenceVersionPurposeConditions, id)
   }
 }
 
+async function _persistLicenceVersionPurposesConditions (
+  trx, updatedAt, licenceVersionPurposeConditions, licenceVersionPurposeId) {
+  for (const licenceVersionPurposeCondition of licenceVersionPurposeConditions) {
```

Comment:
This potentially can be done in a batch but i have followed the previous code pattern.